backend/controllers/materials.controller.ts

```diff
@@ -0,0 +1,21 @@
+import express, {Request, Response } from "express";
+import mongoose from "mongoose";
+import { Material } from "../models/material.model";
+import createMaterial from '../service/materials.service'
+import { getAllMaterials } from "../service/materials.service";
+
+const MaterialController = {
+	Index: async (req: Request, res: Response) => {
+		const materials = await getAllMaterials()
+		res.json({materials})
+    },
+
+	Create: async (req: Request, res: Response) => {
+		const material_data = {name: req.body.name, price: req.body.price, description: req.body.description}
```

Comment:
Nothing wrong with this an alternative way could be to destructure the `req.body` like ths:
`
const {name, price, description} = req.body
`